app/assets/javascripts/admin/bulk_product_update.js.coffee

```diff
@@ -147,7 +147,7 @@ angular.module("ofn.admin").controller "AdminProductEditCtrl", ($scope, $timeout
     if confirm("Are you sure?")
       $http(
         method: "DELETE"
-        url: "/api/products/" + product.id
+        url: "/api/legacy/products/" + product.id
```

Comment:
I'm wondering if there is an opportunity to have kind of builder somewhere like 
```javascript
function apiRoute(path) {
  return "/api/legacy/" + path; // something more robust and tested :p
}
```
But I know it should not changed very often.